tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCheckWidths.java

```diff
@@ -40,4 +42,27 @@ public void TestBasic() {
             assertEquals("errorSubType", Subtype.valueTooWide, err.getSubtype());
         }
     }
+
+    public void TestEnglishWidths() {
+        UnitWidthUtil unitWidthUtils = CheckWidths.UnitWidthUtil.getInstance();
+        CLDRConfig config = CLDRConfig.getInstance();
+        CLDRFile english = config.getEnglish();
+        CheckWidths x = new CheckWidths();
+        CheckWidths.UnitWidthUtil.getInstance();
```

Comment:
Is line 51 superfluous? The returned instance isn't assigned to a variable, and we already have an instance from line 47

Reply:
fixed